troi/playlist.py

```diff
@@ -27,7 +41,7 @@ def _serialize_to_jspf(playlist, created_for=None):
         track["identifier"] = "https://musicbrainz.org/recording/" + str(e.mbid)
         if artist_mbids:
             track["extension"] = {
-                "https://musicbrainz.org#jspf": {
+                PLAYLIST_TRACK_URI_PREFIX: {
                     "artist_mbids" : artist_mbids,
```

Comment:
We should probably add these fields to docs at https://musicbrainz.org#jspf as well.

Reply:
https://wiki.musicbrainz.org/jspf?diff=75246&oldid=74988 we had it but it was named wrong.